easybuild/easyconfigs/l/libharu/libharu-2.3.0-foss-2018a.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Cyprus Institute / CaSToRC, Uni.Lu/LCSB, NTUA
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+# Modified for foss-2016b by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'CMakeMake'
+
+name = 'libharu'
+version = '2.3.0'
+
+homepage = 'http://libharu.org/'
+description = """libHaru is a free, cross platform, open source library for generating PDF files."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [' https://github.com/libharu/libharu/archive/']
+sources = ['RELEASE_%s.tar.gz' % '_'.join(version.split('.'))]
+
+dependencies = [('libpng', '1.6.32')]
```

Comment:
@RvDijk We're going with `libpng` 1.6.34 in easyconfigs that use `2018a` toolchain

Reply:
@boegel libharu is not going to be used, do you still want to include this in this PR?

Reply:
No, let's remove it.